.travis.yml

```diff
@@ -24,10 +24,10 @@ before_install:
 
 script:
   - docker run --name nav2_bash --rm -d -i -t nav2:latest bash -l
-  - docker exec nav2_bash bash -c "colcon test --packages-skip
-      dwb_controller dwb_core nav2_amcl nav2_costmap_2d nav2_dynamic_params 
-      nav2_map_server nav2_navfn_planner nav2_system_tests nav2_tasks 
-      nav2_util nav2_voxel_grid costmap_queue dwb_critics dwb_plugins 
-      nav2_bt_navigator nav2_mission_executor nav2_robot 
-      nav2_simple_navigator nav2_world_model nav_2d_utils"
-  - docker exec nav2_bash bash -c "colcon test-result" 
+  - docker exec nav2_bash bash -c '. /opt/ros/$ROS2_DISTRO/setup.bash &&
+      . /ros2_ws/ros2-linux/setup.bash &&
+      . /ros2_ws/navstack_dependencies_ws/install/setup.bash &&
```

Comment:
The last setup.bash should be the only one you need to source. They include all the setup information that was present in their dependent repos.

Reply:
Thanks.  Addressed.